### PR TITLE
ENG-62 Integrate tweet deleter, screen name checker and stock symbol checker as supervisor services

### PR DESCRIPTION
**Pull request by vitaly-krugl**
ENG-62 Integrated process_tweet_deletions.py and check_twitter_screen_names.py as taurus collector services under supervisord.

ENG-62 Integrated check_company_symbols.py as a taurus collector service under supervisord.

ENG-62 Fixed pylint findings.

### CODE REVIEW

**Comment by vitaly-krugl** [comment]
Ready for review

CC @scottpurdy @oxtopus

**Comment by vitaly-krugl** [comment]
The following stock symbols are now reported as invalid: ACT, DTV, KRFT (merged with heinz), WAG.

The following twitter screen name is now reported as invalid: gdc4systems

**Comment by scottpurdy** [comment]
@oxtopus Do you have time to review this tomorrow?

File: taurus.metric_collectors/taurus/metric_collectors/twitterdirect/process_tweet_deletions.py [hunk 1]
```diff
@@ -144,56 +148,77 @@ def _purgeStaleDeletionRecords(limit):
   return rowIdsToDelete
 
 
+
+def _process():
+  """Perform one iteration of the processing worlflow: purge tweets slated for
```
Comment (oxtopus) [inline taurus.metric_collectors/taurus/metric_collectors/twitterdirect/process_tweet_deletions.py]: "worlflow"?  I assume that's supposed to be "workflow".

**Comment by oxtopus** [comment]
:+1:

**Comment by vitaly-krugl** [comment]
Thanks for reviewing this, @oxtopus. "worlflow" is fixed now.

**Pull request merged**